### ISSUE TITLE
electron_1.crashReporter.start is not a function

### ISSUE THREAD

**Issue by CTianXing** [getsentry/sentry-electron#405]
## Versions + Platform

- [ ] SDK version - `@sentry/electron@v1.5.2`
- [ ] Electron version - `electron@^13.0.0`
- [ ] Platform - `Windows`

## Description
init sentry/electron in the renderer process，there is a wrong:
TypeError: electron_1.crashReporter.start is not a function
    at RendererBackend._installNativeHandler (C:\Users\Administrator\Desktop\project\node_modules\@sentry\electron\dist\renderer\backend.js:80:34)

as know as https://www.electronjs.org/docs/breaking-changes#removed-crashreporter-methods-in-the-renderer-process,
so how to deal this question, thanks

**Comment by timfish** [comment]
`electron_1` suggests you are using a bundler. How are you importing the SDK?

`1.5.2` is quite old. There's currently a v3 beta on npm. 

Without more details of your code/setup or a repository demonstrating the issue I haven't got enough information to know what's going wrong.

**Comment by CTianXing** [comment]
@timfish I have to use 1.5.2， code like this:
```ts
//renderer/main.ts
import * as Sentry from '@sentry/electron'
Sentry.init({
  dsn: 'https://XXX',
  environment: process.env.NODE_ENV,
})

```
just normal usage...

**Comment by timfish** [comment]
What is your bundler configuration?

**Comment by CTianXing** [comment]
it's Vue CLI Plugin Electron Builder, part of it:
```
pluginOptions: {
    electronBuilder: {
      chainWebpackMainProcess: config => {
        // Chain webpack config for electron main process only
      },
      chainWebpackRendererProcess: config => {
        config.module
          .rule('node')
          .test(/\.node$/)
          .use('node-loader')
          .loader('node-loader')
          .end();
        // Chain webpack config for electron renderer process only (won't be applied to web builds)
      },
      outputDir: 'dist',
      mainProcessFile: 'src/main/index.ts',
      // rendererProcessFile: 'src/renderer/main.ts',
      mainProcessWatch: ['src/main'],
      nodeIntegration: true,
      externals: isDev ?
        [] :
        [
          'clipboard',
          'axios',
          'node-ipc',
          '@sentry/electron',
        ],
      files: ['./dist/bundled/**/*'],
      ...
```
I use electron^11.4.10,it is ok, but electron^13.0.0 has a wrong

**Comment by timfish** [comment]
Are you stuck with `1.5.2` because you're using an old version of Sentry On-premises?

**Comment by CTianXing** [comment]
@timfish yes, we all use 1.5.2. There is no problems in mac system, only windows has a wrong.
In @sentry\electron\dist\renderer\backend.js:
![image](https://user-images.githubusercontent.com/22833953/148683538-0d7d7934-37c6-438f-8030-b04b83008eae.png)
due to native reports, crashReporter.start is used.
I set enableNative as false, it's ok. 
I have a question: now, In the current version of electron, is it not necessary to enable crashReporter in the rendering process, even under Windows, Crashpad is also used internally? https://www.electronjs.org/docs/latest/api/crash-reporter

**Comment by timfish** [comment]
It is no longer required to start the crash reporter in the renderer process. This was fixed in `2.2.0` of the SDK.

This means `1.5.2` will only work correctly with Electron <= v11.
